corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -85,7 +89,7 @@ def test_duplicate_client_claim(self):
         self.assertEqual(response.status_code, 200)
         # Dup claim
         response = client.post(url, {'case_id': self.case_id})
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 409)
```

Comment:
I think it did actually change but the tests weren't getting run: https://github.com/dimagi/commcare-hq/pull/11783/files